actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsActor.java

```diff
@@ -102,34 +102,23 @@ private void courseProgressMetricsV2(Request actorMessage) {
     searchDTO.setOffset(offset);
     if (!StringUtils.isEmpty(sortBy)) {
       Map<String, Object> sortMap = new HashMap<>();
-      boolean isNestedSearch = false;
       if (JsonKey.USERNAME.equalsIgnoreCase(sortBy)) {
```

Comment:
At a time user can do sortBy with one attribute only , because same key is replaced with next if condition. so instead of having multiple if use if else condition and move all code for getting sortBy to a separate method which will take sortBy key and return the string.